src/cdomain/value/cdomains/intDomain.ml

```diff
@@ -1776,45 +1774,42 @@ struct
 end
 
 module BigInt = struct
-  include  BI
+  include IntOps.BigIntOps
```

Comment:
Why can't we get rid of this?

Reply:
Because `BigInt.logand` and `BigInt.logor` (from `IntOpsDecorator`) are different from `Z.logand` and `Z.logor`